packages/sdks/snippets/vue/src/views/blueprints/ProductDetails.vue

```diff
@@ -0,0 +1,32 @@
+<template>
+    <div v-if="!productDetails">
+      <p>Loading product details...</p>
+    </div>
+    <div class="product-details-page" v-else>
+      <h1>{{ productDetails.data?.['name'] }}</h1>
+      <img
+        :src="productDetails.data?.['image']"
+        :alt="productDetails.data?.['name']"
+      />
+      <p>{{ productDetails.data?.['collection'].value.data.copy }}</p>
+      <p>Price: {{ productDetails.data?.['collection'].value.data.price }}</p>
+    </div>
+  </template>
+  
+  <script setup lang="ts">
+  import { ref, onMounted } from 'vue'
+  import { fetchOneEntry, typeBuilderContent } from '@builder.io/sdk-vue'
```

Comment:
`typeBuilderContent` is a typo causing the snippet to crash and not build at all.

```suggestion
  import { fetchOneEntry, type BuilderContent } from '@builder.io/sdk-vue'
```